modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf

```diff
@@ -5,6 +5,6 @@ BlueBoard removeAction GVAR(canceledBlu);
 
 [-2, {hint "Blufor has signaled they are ready to begin!";}] call CBA_fnc_globalExecute;
 
-GVAR(bluforIsReady) = True;
+SETMVAR(bluforIsReady, True);
```

Comment:
are these values being sent over the network somewhere?

Reply:
Think it's sorted now?